proto/ibc/applications/transfer/v1/genesis.proto

```diff
@@ -16,4 +17,11 @@ message GenesisState {
     (gogoproto.moretags)     = "yaml:\"denom_traces\""
   ];
   Params params = 3 [(gogoproto.nullable) = false];
+  // total_escrowed contains the total amount of tokens escrowed
+  // by the transfer module
+  repeated cosmos.base.v1beta1.Coin total_escrowed = 4 [
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins",
+    (gogoproto.nullable)     = false,
+    (gogoproto.moretags)     = "yaml:\"total_escrowed\""
```

Comment:
I thought we removed yaml tags 🤔

Reply:
yeah but see [Colin's comment](https://github.com/cosmos/ibc-go/pull/3505/commits/d93e081565928e932c294d96b971fb47133fff83#r1175463103)